dev/appveyor-install-dependencies.ps1

```diff
@@ -129,7 +129,7 @@ $env:PATH = "$env:HADOOP_HOME\bin;" + $env:PATH
 Pop-Location
 
 # ========================== R
-$rVer = "4.0.2"
+$rVer = "4.2.0"
 $rToolsVer = "4.0.2"
```

Comment:
While the tests passed, it failed to download RTools 4.2.0. I reverted the RTools upgrade here for now.